app/scripts/views/visits/index.js

```diff
@@ -41,12 +43,13 @@ define([
     beforeDestroy: function () {
       // stop checking scroll position
       $(window).off('scroll');
+
+      // stop retrying fetch
+      clearTimeout(this.fetchRetryTimer);
```

Comment:
Does the `fetchRetryTimer` always exist?
Me thinks we only set the timer if we hit the `_fetchFailed` branch. But maybe doing `clearTimeout(null || undefined)` is a noop.
```js
    _fetchFailed: function (xhr) {
      // 404 means no results. keep retrying every NO_RESULTS_RETRY_WAIT ms
      if (xhr.status === 404) {
        this.fetchRetryTimer = setTimeout(this._fetch.bind(this), this.FETCH_RETRY_WAIT);
      }
    },
```